lib/createStylelintResult.js

```diff
@@ -118,7 +131,9 @@ module.exports = function(
 			source,
 			deprecations,
 			invalidOptionWarnings,
-			parseErrors,
+			// TODO TYPES check which types are valid? postcss? stylelint?
+			/* eslint-disable-next-line */
+			parseErrors: /** @type {any} */ (parseErrors),
```

Comment:
Is it possible to use object shorthand here? Something like this:

```diff
- parseErrors: /** @type {any} */ (parseErrors),
+ /** @type {any} */ parseErrors,
```

Is it possible when it's a TypeScript file and not JSDoc?

Reply:
same as I described above, it is form of casting in typescript in jsdoc